modules/programs/astroid-config-template.json

```diff
@@ -16,26 +15,6 @@
             "level": "info"
         }
     },
-    "accounts": {
-        "charlie": {
-            "name": "Charlie Root",
-            "email": "root@localhost",
-            "gpgkey": "",
-            "always_gpg_sign": "false",
-            "sendmail": "msmtp -i -t",
-            "default": "true",
-            "save_sent": "false",
-            "save_sent_to": "\/home\/root\/Mail\/sent\/cur\/",
-            "additional_sent_tags": "",
-            "save_drafts_to": "\/home\/root\/Mail\/drafts\/",
-            "signature_separate": "false",
-            "signature_file": "",
-            "signature_file_markdown": "",
-            "signature_default_on": "true",
-            "signature_attach": "false",
-            "select_query": ""
-        }
-    },
```

Comment:
These are cleaned up either way and allowed to see the available options.

Reply:
Hmm, ideally I would like to not have this file at all but I understand it is necessary because astroid cannot currently generate the file without a running X session. So for me the more that can be removed from this file, the better. For information about available options I would direct users to the Astroid manual.

Indeed, maybe more could be removed from it? Perhaps all of these: https://github.com/astroidmail/astroid/blob/439856e3f9c78c198959157ad8fcb1540d456969/src/config.cc#L158-L292 ?